packages/synapse-react-client/src/lib/style/components/_download-list.scss

```diff
@@ -3,7 +3,6 @@
   border-right: 2px solid #dddddd;
   justify-content: center;
   align-items: center;
-  height: 45px;
```

Comment:
Remove since setting the height results in misalignment of DownloadDetails in DownloadConfirmation